### PR TITLE
Add inclusions

### PR DESCRIPTION
**Pull request by HumairAK**
## This introduces a breaking change

- [x] Yes
- [ ] No

## Description

This PR does the following: 

1. Add OCP4 psi spec (#19)
2. Rename dh prod/stage cluster names (as they appear on the ui) 
3. Add server inclusions 
4. Update docs/runbook

### CODE REVIEW

**Comment by HumairAK** [comment]
@harshad16 @goern  -- the inclusion list means that ArgoCD will only see these resources (minus the exclusions list). As such I've included _all_ resources for the quicklab cluster while you guys migrate. Please ensure that that the resource inclusion list satisfies your requirement.

File: vars/prod-vars.yaml [hunk 1]
```diff
@@ -125,7 +125,81 @@ clusters:
...
+  - name: cluster-apps.ocp.prod.psi.redhat.com
+    data:
+      name: apps.ocp.prod.psi.redhat.com
+      namespaces:
+        - dtuchyna-thoth-dev
+        - fmurdaca-thoth-dev
+        - fpokorny-thoth-dev
+        - goern-thoth-dev
+        - jupyterhub-shuels-thoth-integration
+        - kpostlet-thoth-dev
```
Comment (harshad16) [inline vars/prod-vars.yaml:137]: @goern are these namespaces also gonna manage by argo?

**Comment by anishasthana** [comment]
lgtm if Thoth is happy.

File: vars/prod-vars.yaml (hunk 1 above)
Comment (goern) [reply vars/prod-vars.yaml:137]: Hmm, having the option to do so is not bad ;)

**Comment by goern** [comment]
lets activate it!!

**Comment by goern** [comment]
can we do something like `/approve` and tekton gonna merge it?

**Pull request merged**